### PR TITLE
[SAIC-684] Recreate non-accessible image if used by active VM

### PR DESCRIPTION
**Pull request by SVilgelm**

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by ntymtsiv** [comment]
+1, works good

File: cloudferrylib/os/actions/convert_compute_to_image.py [hunk 1]
```diff
@@ -67,6 +66,13 @@ def run(self, info=None, **kwargs):
                     images_body[image_id] = {utl.IMAGE_BODY: {},
                                              utl.META_INFO: {
                                              utl.INSTANCE_BODY: [instance]}}
+                    try:
+                        image_resource.get_ref_image(image_id)
```
Comment (miarmak) [inline cloudferrylib/os/actions/convert_compute_to_image.py]: Does it mean, that we will obtain ALL image's data only for cheking it's availability?

Comment (SVilgelm) [inline cloudferrylib/os/actions/convert_compute_to_image.py]: Yep. We are obtaining ALL image's data to check availability.

Comment (miarmak) [inline cloudferrylib/os/actions/convert_compute_to_image.py]: And what if image is really huge?

Comment (SVilgelm) [inline cloudferrylib/os/actions/convert_compute_to_image.py]: We don't read a body of the image at this step, but we should test this situation anyway.

Comment (miarmak) [inline cloudferrylib/os/actions/convert_compute_to_image.py]: OK, let's test it for sure please.

**Comment by plomakin** [comment]
+1

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**